dispatch-service/src/main/java/de/muenchen/oss/swim/dispatcher/application/usecase/DispatcherUseCase.java

```diff
@@ -125,6 +132,7 @@ protected void processFile(final UseCase useCase, final File file, final Map<Str
         // resolve action
         final DispatchActions action = this.resolveDispatchAction(tags);
         // execute action
+        String destination = action.name();
```

Comment:
At first I found it quite confusing that we always set the `action.name()` to destination, but in case of `DISPATCH` we never use it but overwrite it the the actual destination right away.

Would it not be cleaner / clearer to set `destination` directly in the corresponding case-entry (hence not setting it to `DISPATCH` at all in that case)? Even if that would mean one line of code more. Doing this the variable could even be set to `final` IMO.